src/index.ts

```diff
@@ -135,6 +140,7 @@ type Rectangle = {
  * Get best-looking default options for most images.
  */
 const getDefaultOptions = (): PowerGlitchOptions => ({
```

Comment:
Setting default options per playMode would make sense, particularly adjusting the glitchTimeSpan for play modes hover-*